tutorials/introductory/usage.py

```diff
@@ -449,6 +453,33 @@ def my_plotter(ax, data1, data2, param_dict):
 # numbers or dates.  If you pass 1000 strings Matplotlib will think you
 # meant 1000 categories and will add 1000 ticks to your plot.
 #
+#
+# Additional Axis objects
+# ------------------------
+#
+# Plotting data of different magnitude in one chart may require
```

Comment:
I like this example as two subplots versus two figures.  Please also add links to more info.

Reply:
There are already links to the API reference for [`twinx`](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.twinx.html), [`secondary_xaxis`](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.secondary_xaxis.html#matplotlib.axes.Axes.secondary_xaxis) and [`secondary_yaxis`](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.secondary_yaxis.html#matplotlib.axes.Axes.secondary_yaxis) with each having links to relevant usage examples, so I'd rather not include/duplicate links to the individual examples.

Reply:
My philosophy here is that this should be as good a guide to the rest of the docs as we can give.  Pointing to the API is great, but we should ideally also point to good hooks in the docs where there are more complete explanations of the concepts/techniques.

Reply:
OK, lets add then at the end of the section:  

See [Plots with different scales](https://matplotlib.org/stable/gallery/subplots_axes_and_figures/two_scales.html), [Parasite axis demo](https://matplotlib.org/stable/gallery/axisartist/demo_parasite_axes2.html) and [Secondary Axis](https://matplotlib.org/stable/gallery/subplots_axes_and_figures/secondary_axis.html) for further examples.

Reply:
I amended 07775d574679 to add the links to the examples.